test/e2e/common_test.go

```diff
@@ -566,6 +568,18 @@ func (p *PodmanTestIntegration) CreatePod(options map[string][]string) (*PodmanS
 	return session, session.ExitCode(), session.OutputToString()
 }
 
+func (p *PodmanTestIntegration) TestPodOptions() bool {
+	exampleOptions := entities.ContainerCreateOptions{PID: "ns:/proc/self/ns"}
+
+	podOptions := entities.PodCreateOptions{}
+	err := common.MapOptions(&exampleOptions, &podOptions)
+	if err != nil {
+		return false
+	}
+
+	return podOptions.Pid == "ns:/proc/self/ns"
+}
```

Comment:
This should be a unit test in cmd/podman/common/create_test.go ant not a integration test.
It should also test more than one field.

Reply:
@Luap99 do you mind if I use reflect to check if the values are equal for all of the entries or should I go through 1 by 1 manually and check

Reply:
Sure you can use reflect for testing. You don't have to test every field. My main concern is that this should be part of the unit tests and not e2e tests.